scripts/apply-updates

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright (C) 2019  Red Hat, Inc.
+#
+# Download and extract the updates image.
+#
+# How to apply the updates image?
+#   apply-updates http://myserver/updates.img
+#
+# How to run Anaconda with updates?
+#   anaconda
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+
+URL="$1"
+
+if [[ -z "${URL}" ]]; then
+    echo "Specify the location of the updates image." 1>&2
+    exit 1
+fi
+
+# Clear /tmp/updates, so Anaconda can set it up again.
+rm -rf /tmp/updates
+
+# Download and extract the updates image.
+(cd / ; curl -f "${URL}" | gzip -dc  | cpio -idu )
```

Comment:
For future extension it should also support tarball image files which are supported by Anaconda if I'm not mistaken. But this is fine for now.